src/components/Button/index.tsx

```diff
@@ -19,15 +18,14 @@ export const LogInButton: React.FC<Props> = ({ action, title }) => {
 
 const styles = StyleSheet.create({
   button: {
-    width: 200,
-    height: 40,
-    margin: 10,
-    padding: 10,
+    width: '50%',
```

Comment:
maybe let's use flex here? We don't really use percentage based width much in RN